web/pom.xml

```diff
@@ -100,6 +100,18 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.jaxrs</groupId>
```

Comment:
If you need help on this drop me a line. I dont know if you remember,  but some time ago I did a POC on this. https://github.com/quilombodigital/orbit/commits/bridge